homeassistant/auth/auth_store.py

```diff
@@ -17,6 +17,8 @@
 from .permissions import PermissionLookup, system_policies
 from .permissions.types import PolicyType
 
+# mypy: disallow-any-generics
```

Comment:
I rather would make a local exception instead of disabling it for the whole file.

Reply:
Note that disallowing here means we're making things _stricter_ for this file compared to our baseline options. That cannot be done with a local per-line or so exception.

We _could_ make our baseline options stricter by doing this setting there, but that'd require sprinkling a lot of those local or file based exceptions around the tree. I think we are not ready for that yet.

Reply:
Oh right! 🤦 I've been sleeping reviewing this allow vs disallow. Yeah, that makes sense 👍